extensions/api/data-management/policydefinition/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/policy/PolicyApi.java

```diff
@@ -18,19 +18,19 @@
 import io.swagger.v3.oas.annotations.tags.Tag;
 import jakarta.validation.Valid;
 import org.eclipse.dataspaceconnector.api.query.QuerySpecDto;
-import org.eclipse.dataspaceconnector.policy.model.Policy;
+import org.eclipse.dataspaceconnector.policy.model.PolicyDefinition;
 
 import java.util.List;
 
 @OpenAPIDefinition
 @Tag(name = "Policy")
 public interface PolicyApi {
```

Comment:
Will be more time consuming, but it would be better to add a policydefinition api module and deprecate the policy one, will be more easy for users to handle the breaking change

Reply:
as far as I am aware, there was a general understanding that breaking changes are "allowed" without deprecation. In this case it would probably mean to maintain the "old" and new policy DTOs.

What we could do as next-best-thing is create a git tag right before merging this PR and publishing the artifacts, to give people a somewhat up-to-date version

Reply:
ok, my proposal would be to rename the api to `PolicyDefinitionApi`, the basepath to `policydefinitions` and the controller class accordingly. the module is already called `policydefinition` so.. effort earned :)

Reply:
I would like to avoid this unless there is a clear and present requirement from any of the projects to have the "old" policy API in the latest upstream `main`.

we're miles away from a "final" version, so I think the expectation from the general public to have stable APIs is not really justified.

Reply:
@paullatzelsperger forget about the "breaking change", this comment could be summarized as:
- `PolicyApi` class should be renamed to `PolicyDefinitionApi` and the related controller as well
- the api `@Path` should be moved from `/policies` to `/policydefinitions`

Reply:
done. also renamed the extension to `PolicyDefinitionApiExtension` to keep consistency.